examples/react-load/react/docker/docker-compose.yaml

```diff
@@ -3,8 +3,8 @@ services:
 
   # Collector
   collector:
-    image: omnition/opentelemetry-collector-contrib:0.2.8
-    command: ["--config=/conf/collector-config.yaml", "--log-level=DEBUG"]
+    image: otel/opentelemetry-collector-contrib:0.40.0
```

Comment:
Did you try this with collector 40? Our exporters are not on the latest proto so I want to make sure it works.

Reply:
yeah, I just tried it again on a clean repo